src/components/TasksTable/index.jsx

```diff
@@ -31,8 +35,18 @@ const sorted = pipe(
   tags: {
     whiteSpace: 'nowrap',
   },
+  dropdown: {
+    minWidth: 200,
+  },
+  filter: {
+    padding: `0px ${3 * theme.spacing.unit}px`,
```

Comment:
On smaller screens this will not align with "Bugs & Issues" because the padding changes with screen size. To fix this you can do:

```jsx
filter: {
  ...theme.mixins.gutters(),
},
```